### DISCUSSION TITLE
New Exclusive Themes for BigLinux - Turing Smart Screen Python

### DISCUSSION THREAD

**Discussion by ruscher** [mathoudebine/turing-smart-screen-python#402] [Themes]
I've crafted two new themes for the Turing Smart Screen Python on BigLinux. To make things easy, I've packaged these themes for automatic installation.

![image](https://github.com/mathoudebine/turing-smart-screen-python/assets/355281/98a1659f-babc-4d8b-8013-40450127645f)
![image](https://github.com/mathoudebine/turing-smart-screen-python/assets/355281/5c4b4620-6262-4fae-b2f7-0530d0b53f33)

Icon: 
![biglinux-grub-restore](https://github.com/mathoudebine/turing-smart-screen-python/assets/355281/85fc7174-a1f1-4393-99c8-b02b8d34768e)

https://github.com/ruscher/turing-smart-screen/tree/main/turing-smart-screen/usr/share/turing-smart-screen/res/themes

**Comment by DJRoby19** [comment]
Really nice compliments
only thing I would enlarge the characters
